src/shellbar.scss

```diff
@@ -1,2 +1,234 @@
-@import "../scss/components/shellbar";
-@import "../scss/fonts/72";
+@import "./settings";
+@import "./mixins";
+@import "./functions";
+@import "./shellbar/product-menu";
+@import "./shellbar/product-switcher";
+@import "./shellbar/search-input";
+
```

Comment:
should shellbar's subcomponents just be promoted to top level? the datepicker and timepicker are (or should be) is just a wrappers around other subcomponents as well.

Reply:
We can, either way is fine with me 🤷‍♀ 

Datepicker and timepicker don't even have files since there's no styles associated with them